cordapp-example/workflows-kotlin/src/test/kotlin/com/example/test/flow/IOUPaybackFlowTests.kt

```diff
@@ -60,11 +59,35 @@ class IOUPaybackFlowTests {
     }
 
     @Test
-    fun `flow rejects invalid UUID string`() {
-        val flow = PaybackFlow.Initiator("123", iou.value)
+    @Ignore
+    fun `throw when invalid UUID string`() {
+        val flow = PaybackFlow.Initiator("123", iouInput.value)
+        val future = lender.startFlow(flow)
+        network.runNetwork()
+
+        assertFailsWith<IllegalArgumentException> { future.getOrThrow() }
+    }
+
+    @Test
+    fun `throw when value doesn't match IOU`() {
+        val flow = PaybackFlow.Initiator(iouInput.linearId.toString(), iouInput.value - 1)
```

Comment:
Can you not take the value from the `iouInput` instead of asking the caller to pass this redundant info?